Atom feed support

ingilniero opened the pull request.

Commit:
Add Atom Feeds

Commit:
Add test for AtomFeedsController

Commit:
Add discovery_tag to display that a RSS feed is available

Commit:
Add support to show both Events and Blogpost in the feed

mariochavez commented:
Could you add a blog.rss as well, looks like there is a feedburder for it, but we have no access to it.

mariochavez commented:
Also cache the query - merge with blog posts/events expire the cache twice a day, to avoid hitting the DB every time someone check the feeds

mariochavez commented:
Rename AtomFeedsController to be just FeedsController, add support to have it only respond to Atom, Rss. Rss endpoint should be blog.rss

Commit:
Add RSS support

Change controller name to FeedsController
Change endpoint to "blog"
Add caching to posts and events queries
Update tests
